doc/compressed_transactions.md

```diff
@@ -0,0 +1,178 @@
+| VarInt Flag      | 2 Bits    | 1, 2 or 3 indicate literal values. 0 indicates that the value will be encoded in a later VarInt. |
+| Bitstream VarInt | 1+ Bytes  | 7-bit little-endian encoding, with each 7-bit word encoded in a byte. The highest bit of each byte is 1 if more bytes follow, and 0 for the last byte. |
+| VLP-Bytestream   | 2+ Bytes  | A VarInt Length Prefixed Bytestream, Has a VarInt prefixed to determine the length. |
+
+### 3.2 General Schema
+
+| Name                           | Width           | Description |
+|--------------------------------|-----------------|-------------|
+| Transaction Metadata           | 7 Bits    | Information on the structure of the transaction. See Section 3.3. |
+| Minimum Blockheight            | 1-5 Bytes | A Bitstream VarInt containing the Minimum Blockheight of which the transaction locktime and input blockheights are given as offsets. |
+| Version                        | 0-5 Bytes | An optional VarInt containing the transactions version. |
+| Input Count                    | 0-5 Bytes | An optional VarInt containing the transactions input count. |
+| Output Count                   | 0-5 Bytes | An optional VarInt containing the transactions output count. |
+| LockTime                       | 0-5 Bytes | An optional VarInt containing the transaction LockTime if its non zero. |
+| Input Metadata+Output Metadata | 1+ Bytes  | A Bitstream Encoding containing metadata on all the inputs and then all the outputs of the transaction. For each input see Section 3.4, for each output see Section 3.5. |
+| Input Data                     | 66+ Bytes | See Section 3.6 for each input. |
+| Output Data                    | 3+ Bytes  | See Section 3.7 for each output. |
+
+For the four VarInts listed above we could use a more compact bit encoding for these but they are already a fall back for the bit encoding of the Transaction Metadata.
+
+### 3.3 Transaction Metadata
+
+| Name         | Width  | Description |
+|--------------|--------|-------------|
+| Version      | 2 Bits | A VarInt Flag for the transaction version. |
+| Input Count  | 2 Bits | A VarInt Flag for the transaction input count. |
+| Output Count | 2 Bits | A VarInt Flag for the transaction output count. |
+| LockTime     | 1 Bit  | A Boolean to indicate if the transaction has a LockTime. |
+
+### 3.4 Input Metadata
+
+| Name                 | Width  | Description |
+|----------------------|--------|-------------|
+| Compressed Signature | 1 Bit  | A Flag to determine the state of the signatures compression, For P2TR: 1 for keyspend, 0 for scriptspend; For P2SH: 0 for p2sh, 1 for p2sh-wpkh. |
+| Standard Hash        | 1 Bit  | A Flag to determine if this Input's Signature Hash Type is standard (0x00 for Taproot, 0x01 for Legacy/Segwit). |
+| Standard Sequence    | 2 Bits | A VarInt Flag for the inputs sequence, Encode literal values as follows: 1 = 0x00000000, 2 = 0xFFFFFFFE, 3 = 0xFFFFFFFF. |
```

Comment:
In 6a117eac0af155610bb7bcfa30b6b878ca9628f4:

`,` should be `.`. This occurs throughout the doc.

Reply:
Fixed, Thanks!